lib/main/atom/autoCompleteProvider.ts

```diff
@@ -88,6 +89,53 @@ export class AutocompleteProvider implements Provider {
     return suggestions
   }
 
+  formatImportPath(sourcePath: string): string {
+    sourcePath = sourcePath.replace(/\.d$/, ""); // tsconfig.removeExt can convert d.ts file path into the filepath end with `.d`;
+    sourcePath = sourcePath.replace(/.*\/node_modules\//, "");
+    return sourcePath;
+  }
+
+  async getPathSuggestions(
+    prefix: string,
+    location: FileLocationQuery): Promise<SuggestionWithDetails[]> {
+    const client = await this.clientResolver.get(location.file)
+    const projectInfo = await client.executeProjectInfo({
+      file: location.file,
+      needFileNameList: true
+    })
+    const filePaths = projectInfo.body!.fileNames!.filter(p => p != location.file);
+    var files: {
+      name: string;
+      relativePath: string;
+      fullPath: string;
+    }[] = []
+    var sourceDir = path.dirname(location.file);
+    filePaths.forEach(p => {
+      let relativePath = path.relative(sourceDir, p).split('\\').join('/');
```

Comment:
Is this meant to convert from windows paths to unix? If so, there are specialized functions in `path` module for this.